interfaces/cython/cantera/utils.pyx

```diff
@@ -31,6 +31,10 @@ def add_directory(directory):
     """ Add a directory to search for Cantera data files. """
     CxxAddDirectory(stringify(directory))
 
+def get_data_directories(sep=':'):
```

Comment:
On Windows, if you call `getDataDirectories` with `:` as the separator, it will return a string like:
```
C:\src\anaconda\envs\py33\lib\site-packages\cantera\data:'C:\Program Files\Cantera\data
```
Which Python will then split as:
```python
['C',
 '\\src\\anaconda\\envs\\py33\\lib\\site-packages\\cantera\\data',
 'C',
 '\\Program Files\\Cantera\\data']
```